sql/core/src/main/scala/org/apache/spark/sql/execution/python/PythonArrowOutput.scala

```diff
@@ -182,15 +192,24 @@ class ArrowOutputProcessorImpl(reader: ArrowStreamReader, pythonMetrics: Map[Str
   }
 }
 
-class SliceArrowOutputProcessorImpl(
```

Comment:
No biggie but there's `BatchedPythonArrowInput`.  Would it make more sense to have `BatchedPythonArrowOutput`?

Reply:
We could, although I am not sure when we only want `BasicPythonArrowOutput` without batched output feature. The difference is for a `BatchedPythonArrowOutput`, we may have an abstract method like `getArrowOutputProcessor` in `PythonArrowOutput` and let `BasicPythonArrowOutput` and `BatchedPythonArrowOutput` to implement it.

I can do it a bit refactoring to extract it in a follow up if you want.